crates/bevy_render/src/extract_component.rs

```diff
@@ -225,13 +226,13 @@ fn extract_components<C: ExtractComponent>(
 fn extract_visible_components<C: ExtractComponent>(
```

Comment:
Same here. Before, the nuke would handle culling our previously-visible lights but now we don't have that "luxury"